### ISSUE TITLE
When video pause, how can i dismiss the ui "more video"?

### ISSUE THREAD

**Issue by laiyi55** [PierfrancescoSoffritti/android-youtube-player#892]

**Comment by Serkali-sudo** [comment]
#### To hide 'More Videos' section  

Go to [ayp_youtube_player.html#L116](https://github.com/PierfrancescoSoffritti/android-youtube-player/blob/a9b5a70292b00f7b2f61d79d2debea22462a0c85/core/src/main/res/raw/ayp_youtube_player.html#L116) add this function and call this function from inside onReady state and it will hide the popup

```js
function hideMoreVideosPopup() {
            setInterval(() => {
                const playerIFrame = document.querySelector("iframe");

                if (!playerIFrame) {
                    return;
                }

                const frameDoc = playerIFrame.contentDocument;

                if (!frameDoc) {
                    return;
                }

                 const overlayContainer = frameDoc.getElementsByClassName("ytp-pause-overlay-container")[0];
                  if (!overlayContainer) {
                    return;
                }
                 overlayContainer.style.display = 'none';

            }, 100);
        }

```

**Comment by laiyi55** [comment]
thanks a lot，but is it legal？how about title and logo？I find issue to hide title, but no method to hide logo

**Comment by laiyi55** [comment]
And if I use sdk, is there any method to change ayp_youtube_player.html?

**Comment by Serkali-sudo** [comment]
I think unless you don't hide the YouTube logo at the bottom corner it should be legal to hide anything else (eg : you can hide title,channel,etc..)
But i am not sure you should read the [official documentation](https://developers.google.com/youtube/iframe_api_reference) to be sure.

And to be able to edit this library in your project you can just add this library as a module in Android Studio.

**Comment by laiyi55** [comment]
<img width="927" alt="image" src="https://user-images.githubusercontent.com/25626719/199452692-9c1d4499-ea25-4cf9-b23e-751b55dad8c5.png">
hi~I finid this way to change quelity, did it work? should getQualityList after onReady callback?

**Comment by Serkali-sudo** [comment]
Yes it works. But i don't recommend you to call `getAvailableQualityLevels` from onReady callback because onReady is calling just once after you load the player view and if you load another video onReady will not be called again.You can call it from somewhere else for example PLAYING_STATE

**Comment by PierfrancescoSoffritti** [comment]
FYI: altering the UI of the IFrame player UI in any way might give you issue with YouTube terms of service if you're publishing the app on the PlayStore.

**Comment by laiyi55** [comment]
understood! thanks a lot

**Comment by rachit1797** [comment]
Can I use this with custom view. And how to use [ayp_youtube_player](https://github.com/PierfrancescoSoffritti/android-youtube-player/blob/a9b5a70292b00f7b2f61d79d2debea22462a0c85/core/src/main/res/raw/ayp_youtube_player.html#L116) with the Iframe player I am currently using 
 val options: IFramePlayerOptions =
            IFramePlayerOptions.Builder().controls(0).rel(0).build()
